### DISCUSSION TITLE
Authentication fail - Slack

### DISCUSSION THREAD

**Discussion by divvetec** [outline/outline#7334] [Self Hosting]
Hello! Can anyone help me? 

I'm leaning about the open source community and had some issues trying to install Outline on docker with portainer.

The app aparently its functioning, but i cant authenticate with slack, google or other method.

See the app:
https://docs.divve.co

Below is the code I put in the portainer stack:

`version: "3.7"

services:
  outline:
    image: outlinewiki/outline:0.75.0

    volumes:
      - outline_data:/var/lib/outline/data
      
    networks:
      - my_net
    
    environment:
      ## Dados de acesso
      - URL=https://docs.divve.co
      - FORCE_HTTPS=true

      ## Desativar novos cadastros
      - LIMITS_INVITATION_REQUIRED=false

      # IDIOMA PADRÃO
      - DEFAULT_LANGUAGE=pt_BR

      ## Secrets Keys
      - SECRET_KEY=b8b4d9e2075696d5e40119395de06f9644bad47ad092844520bde61c9b165b56
      - UTILS_SECRET=8dfedf9e4d219e5f457b6bd4a1610f6499b0250a53c8e10558a0377d95d20a8b
      
      ## Dados Postgres
      - DATABASE_URL=postgres://postgres:@@@@postgres:5432/outline
      - DATABASE_URL_TEST=postgres://postgres:@@@@postgres:5432/outline_test
      - PGSSLMODE=disable
      
      ## Dados Redis
      - REDIS_URL=redis://redis:6379/3
      
      ## Dados sobre Armazenamento
      - FILE_STORAGE=local
      - FILE_STORAGE_LOCAL_ROOT_DIR=/var/lib/outline/data
      - FILE_STORAGE_UPLOAD_MAX_SIZE=26214400
      - FILE_STORAGE_IMPORT_MAX_SIZE=18874368
      - FILE_STORAGE_WORKSPACE_IMPORT_MAX_SIZE=18874368
      
      ## LOGIN COM O GOOGLE
      # - GOOGLE_CLIENT_ID=@@@
      # - GOOGLE_CLIENT_SECRET=@@@@@
      # - NEXT_PUBLIC_GOOGLE_API_KEY=@@@@

      ## LOGIN COM O SLACK
      - SLACK_CLIENT_ID= @@@
      - SLACK_CLIENT_SECRET= @@@
      
      ## Dados do SMTP
      - SMTP_FROM_EMAIL=suporte@divve.co ## Email SMTP
      - SMTP_REPLY_EMAIL=suporte@divve.co ## Email SMTP
      - SMTP_USERNAME=@@@ ## Usuario SMTP
      - SMTP_PASSWORD=@@@ ## Senha SMTP
      - SMTP_HOST=br.greedo1090.com.br ## Host SMTP
      - SMTP_PORT=465 ## Porta SMTP
      - SMTP_SECURE=true ## Se a porta for 465 ou 25 = true || Qualquer outra = false

      ## Outras configurações
      - RATE_LIMITER_ENABLED=true
      - RATE_LIMITER_REQUESTS=1000
      - RATE_LIMITER_DURATION_WINDOW=60
      - ENABLE_UPDATES=false
      - WEB_CONCURRENCY=6
      - DEBUG=http
      - LOG_LEVEL=info
      
    dns:
      - 1.1.1.1
    
    deploy:
      mode: replicated
      replicas: 1
      placement:
        constraints:
          - node.role == manager
      resources:
        limits:
          cpus: "1"
          memory: 1024M
      labels:
      - traefik.enable=true
      - traefik.http.routers.outline.rule=Host(`docs.divve.co`)
      - traefik.http.routers.outline.entrypoints=websecure
      - traefik.http.routers.outline.tls=true
      - traefik.http.routers.outline.tls.certresolver=letsencryptresolver
      - traefik.http.routers.outline.service=outline
      - traefik.http.services.outline.loadbalancer.server.port=3000
      - traefik.http.middlewares.test-redirect.redirectscheme.scheme=https
      - traefik.http.routers.outline.middlewares=test-redirect@docker

volumes:
  outline_data:
    external: true
    name: outline_data
    
networks:
  my_net:
    external: true
    name: my_net`

I checked the credentials several times and they are correct.

Postgree and Redis are installed and configured on the same network. Where am I going wrong?

**Comment by tommoor** [comment]
I don't believe you've described the actual error or the errors logs involved?

**Reply by tommoor** [reply]
@divvetec you posted secret keys in the message above, you should immediately revoke them for they are scraped and abused.
